xml/installation-installation-configure-deployer-repos.xml

```diff
@@ -0,0 +1,487 @@
+    Pool repositories are provided by the &scc;. They contain all updates and
+    patches for the products and extensions. To make them available for
+    &cloud; they need to be mirrored from the &scc;. Since their content is
+    regularly updated, they must be kept in synchronization with &scc;. For
+    these purposes, &suse; provides either the &smtool; (&smt;) or the
+    &susemgr;.
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="sec.depl.adm_conf.repos.product">
+  <title>Copying the Product Media Repositories</title>
+  <para>
+   The files in the product repositories for &sls; and &cloud; do not
+   change, therefore they do not need to be synchronized with a remote
+   source. If you have installed the product media from a downloaded ISO image,
+   the product repositories will automatically be made available to the client
+   nodes and these steps can be skipped. Similarly, if you have configured the
+   &lcm; node as an SMT mirror, the product repositories will automatically be
+   made available to the client nodes and these steps can be skipped.
+   Otherwise, it is sufficient to either copy the data (from a remote host or
+   the installation media), to mount the product repository from a remote
+   server via <literal>NFS</literal>, or to loop mount a copy of the
+   installation images.
+   </para>
+   <para>
+    The following product media needs to reside in the specified
+    directories:
+   </para>
+   <table>
+    <title>Local Product Repositories for &cloud;</title>
+    <tgroup cols="2">
+     <colspec colnum="1" colname="1" colwidth="40*"/>
+     <colspec colnum="2" colname="2" colwidth="60*"/>
+     <thead>
+      <row>
+       <entry>
+        <para>
+         Repository
+        </para>
+       </entry>
+       <entry>
+        <para>
+         Directory
+        </para>
+       </entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>
+        <para>
+         &productname; &productnumber; DVD #1
+        </para>
+       </entry>
+       <entry>
+        <para>
+         <filename>&www_dir;/repos/Cloud</filename>
+        </para>
+       </entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </table>
+   <para>
+    The data can be copied by a variety of methods:
+   </para>
+   <variablelist>
+    <!-- sec.depl.adm_conf.repos.product.media -->
+    <varlistentry>
+     <term>Copying from the Installation Media</term>
+     <listitem>
+     <para>
+      We recommended using <command>rsync</command> for copying. If the
+      installation data is located on a removable device, make sure to mount
+      it first (for example, after inserting the DVD1 in the &admserv; and
+      waiting for the device to become ready):
+     </para>
+     <bridgehead renderas="sect4">&cloudos; DVD#1</bridgehead>
+     <screen>mkdir -p &www_dir;/repos/&sle_repo;-Pool
+mount /dev/dvd /mnt
+rsync -avP /mnt/ &www_dir;/repos/&sle_repo;-Pool/
```

Comment:
hmm, not quite the iso is in susetags format, the -Pool channel in repomd. so copying it to -Pool will not work if you switch it then to SCC later (because the repositories detect susetags and for pool it is then later repomd and zypper will refuse to run)

Reply:
Hmm I see. Should I just omit mentioning the SLES media? (It's mentioned in http://docserv.nue.suse.com/documents/Cloud7/suse-openstack-cloud-deployment/html/cha.depl.repo_conf.html#sec.depl.adm_conf.repos.product) Or I can bring the path recommendation back into alignment with that doc and recommend using the install/ directory (will have to make some code changes for that too).

Reply:
Fixed this, was a misunderstanding on my part about what this was for.